src/components/PostDetails/PostDetails.tsx

```diff
@@ -1,45 +1,97 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { deletePostComments, getPostComments } from '../../api/comments';
 import { NewCommentForm } from '../NewCommentForm';
 import './PostDetails.scss';
 
-export const PostDetails: React.FC = () => (
-  <div className="PostDetails">
-    <h2>Post details:</h2>
+interface Props {
+  post: Post;
+}
 
-    <section className="PostDetails__post">
-      <p>sunt aut facere repellat provident occaecati excepturi optio</p>
-    </section>
+export const PostDetails: React.FC<Props> = ({ post }) => {
+  const [comments, setComments] = useState<Comment[] | null>(null);
+  const [hideComments, setHideComments] = useState(false);
 
-    <section className="PostDetails__comments">
-      <button type="button" className="button">Hide 2 comments</button>
+  const deleteComment = async (id: number) => {
+    await deletePostComments(id);
+  };
 
-      <ul className="PostDetails__list">
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>My first comment</p>
-        </li>
-
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
+  const getComments = async (postId: number) => {
+    try {
+      const commentsFromS = await getPostComments(postId);
+
+      setComments(commentsFromS);
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.log(error, 'Request failed');
+    }
+  };
+
+  const updateComments = async (id: number, postId: number) => {
+    await deleteComment(id);
+    await getComments(postId);
+  };
+
+  useEffect(() => {
+    getComments(post.id);
```

Comment:
You should receive only post id in this component and call getPostDetails method to show post details info before the comments link